jumpcloud_bootstrap_template.sh

```diff
@@ -710,7 +750,7 @@ if [[ ! -f $DEP_N_GATE_UI ]]; then
 
         WINDOW_TITLE='Set a password'
         PASSWORD_TITLE="Please set a password"
-        PASSWORD_TEXT='Your password must be 8 characters long and contain at least one number, upper case character, lower case character, and special character. \n The longer the better!'
+        PASSWORD_TEXT="Your password must be $minlength characters long and contain at least one number, upper case character, lower case character, and special character. \n The longer the better!"
```

Comment:
This change should have been made ages ago, it's just a variable now but Though testing I can confirm this change shows up here and below where we actually prompt a user for input.
![Screen Shot 2023-07-26 at 12 54 09 PM](https://github.com/TheJumpCloud/MDM-Prestage-User-Enrollment/assets/54448601/f7529bb1-0d47-42fc-91a7-b77809785b1a)